### DISCUSSION TITLE
make it so you get greentext when you kill no matter if the objective is cloned/respawned

### DISCUSSION THREAD

**Discussion by corp-0** [unitystation/unitystation#8183] [Ideas]
### Description
It seems like many people don't like the idea of letting antagonists get rid of bodies when they are their objective. It was suggested to maybe consider the objective done at the first kill, without taking in consideration if the objective was cloned or respawned.

**Comment by wraith-54321** [comment]
maybe for now but once ghosts have things to do I think this should be changed back

**Reply by AtnerNT** [reply]
> maybe for now but once ghosts have things to do I think this should be changed back

Agreed, I think a big reason why people don't like getting rid of bodies is that they feel bad that the person now just has nothing to do and will be bored- or at the very least- the things they can do are no where near as interesting as just being on station. However once ghost have alot more things to do, and the existing stuff is expanded, I think people will feel far less bad about ridding bodies so it can be put back.

**Comment by CdrCross** [comment]
I agree with this idea, though it should definitely be easy to toggle on-off on a server by server basis.
